packages/one-app-bundler/__tests__/webpack/loaders/common.spec.js

```diff
@@ -69,6 +69,24 @@ describe('Common webpack loaders', () => {
       expect(purgeCssLoader()).toMatchSnapshot();
     });
 
+    it('should include any additional configured options', () => {
+      const purgecss = {
+        paths: ['foo', 'bar'],
+        extractors: [{
+          extractor: 'purgeJs',
+          extensions: ['js'],
+        }],
+        fontFace: true,
+        keyframes: true,
```

Comment:
Could you please add this to the README.md

Reply:
added in https://github.com/americanexpress/one-app-cli/pull/61/commits/aabe8f09c1e106c906c1d3c4897423f54ceadfc8 🌮